docs/HowTo/Backup/Backup.md

```diff
@@ -1,57 +1,53 @@
-description: Backing up and restoring Besu   
+description: Backing up and restoring Besu
 <!--- END of page meta data -->
 
-# Backups 
+# Backups
 
-In a decentralized blockchain, data is replicated on other nodes so it cannot be lost. However, backing up 
-configuration and data ensures a smoother recovery.  
+In a decentralized blockchain, data replicates between nodes so it is not lost, but backing up
+configuration and data ensures a smoother recovery from corrupted data or other failures.
 
-## Genesis File 
+## Genesis file
 
-The genesis file for a network must be accessible on every node. We recommend the genesis file is 
-stored under source control. 
+The genesis file for a network must be accessible on every node. We recommend the genesis file is
+stored under source control.
 
-## Data Backups
+## Data backups
 
-If installed locally, the default data location is the directory in which Besu is installed.
+If installed locally, the default data location is the Besu installation directory.
 
-We recommend mounting a [separate volume to store data](../Get-Started/Run-Docker-Image.md#starting-besu). 
-Use the [`--data-path`](../../Reference/CLI/CLI-Syntax.md#data-path) command line option to pass the path to Besu. 
+We recommend mounting a
+[separate volume to store data](../Get-Started/Run-Docker-Image.md#starting-besu). Use the
+[`--data-path`](../../Reference/CLI/CLI-Syntax.md#data-path) command line option to pass the path
+to Besu.
 
-The default data location is the directory in which Besu is installed
-or `/opt/besu/database` if using the [Besu Docker image](../../HowTo/Get-Started/Run-Docker-Image.md).
+The default data location is the Besu installation directory, or `/opt/besu/database` if using the
+[Besu Docker image](../../HowTo/Get-Started/Run-Docker-Image.md).
+Having some data reduces the time to synchronise a new node. You can perform periodic backups of
```

Comment:
Should there be a line between these two paragraphs?

Reply:
Good pick up.